docker-entrypoint.sh

```diff
@@ -12,11 +12,11 @@
 
 set -eu
 
-raw_cloud_config_dir="/usr/code/cloudconfig/openstack/latest"
-raw_cloud_config_path="$raw_cloud_config_dir/user_data"
+RAW_CLOUD_CONFIG_DIR="/usr/code/cloudconfig/openstack/latest"
+RAW_CLOUD_CONFIG_PATH="${RAW_CLOUD_CONFIG_DIR}/user_data"
```

Comment:
https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization